tests/rkt_run_pod_manifest_test.go

```diff
@@ -504,13 +518,12 @@ func TestPodManifest(t *testing.T) {
 			continue
 		}
 
-		j := 0
-		for name, patches := range tt.images {
-			imageFile := patchTestACI(name, patches...)
+		for j, v := range tt.images {
+			imageFile := patchTestACI(v.name, v.patches...)
```

Comment:
For the test `rkt-test-run-pod-manifest-app.aci`, patchTestACI will be executed twice and give the same imageFile. `defer os.Remove` will be executed twice on the same file. But I guess it's fine.